extensions/ql-vscode/src/view/results/result-tables.tsx

```diff
@@ -78,6 +78,52 @@ function renderResultCountString(resultSet: ResultSet): JSX.Element {
   );
 }
 
+function getInterpretedTableName(interpretation: Interpretation): string {
+  return interpretation?.data.t === "GraphInterpretationData"
```

Comment:
Minor: Is `?` necessary? According to the type information, `interpretation` can never be undefined.

```suggestion
  return interpretation.data.t === "GraphInterpretationData"
```